adhocracy4/emails/__init__.py

```diff
@@ -15,6 +15,9 @@ class EmailBase:
     fallback_language = 'en'
     for_moderator = False
 
+    def get_reply_to(self):
+        return []
```

Comment:
i'd prefer to return `None` by default, as this is the current default value for EmailMultiAlternatives

https://github.com/django/django/blob/1.8.18/django/core/mail/message.py#L407